src/sentry/utils/snuba.py

```diff
@@ -1343,11 +1349,12 @@ def aliased_query_params(
             derived_columns.append(aggregation[2])
 
     if conditions:
-        column_resolver = (
-            functools.partial(condition_resolver, dataset=dataset)
-            if condition_resolver
-            else resolve_func
-        )
+        if condition_resolver:
+            column_resolver: Callable[[Any], Callable] = functools.partial(
```

Comment:
@asottile-sentry I have no idea why this works 😅

Reply:
yeahhh I think this is wrong -- or at least the parameter in the original function isn't correct

I think if you move this annotation to the function signature it'll fix it